lib/zold/remotes.rb

```diff
@@ -187,7 +190,7 @@ def iterate(log, farm: Farm::Empty.new)
               network: @network
             )
             idx += 1
-            raise 'Took too long to execute' if (Time.now - start).round > @timeout
+            raise 'Took too long to execute' if (Time.now - start).round > timeout
```

Comment:
@emilianodellacasa have you heard of `Timeout.timeout`?

Reply:
Actually this is something I merged from another issue, I didn't do that :-)